gwr/sel_bw.py

```diff
@@ -207,24 +201,117 @@ def search(self, search='golden_section', criterion='AICc', bw_min=0.0,
         self._bw()
 
         return self.bw[0]
+            
+    #hold it for allowing lat-lons in next PR
+    #_haversine formula to calculate distance
+    def _haversine(lat1, lon1, lat2, lon2):
+        R = 6371400 # Earth radius in meters
+        dLat = radians(lat2 - lat1)
```

Comment:
Also, where is this `raidans` function coming from? it hasn't been imported and I don't see it defined?